portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TextInput.path

```diff
@@ -165,6 +165,11 @@
 	<td>//input[contains(@value,'${key_unnamedTitle}')]</td>
 	<td></td>
 </tr>
+<tr>
```

Comment:
We would recommend to not add duplicate path, the description is a textarea, we can remove this and try to use the existing TextInput#ANY_ID